src/app/settings/settings.html

```diff
@@ -93,6 +93,9 @@ <h1 ng-i18next="Settings.Settings"></h1>
             <input type="radio" ng-model="vm.settings.itemSort" value="primaryStat"><span ng-i18next="Settings.SortPrimary"></span></label>
           <br>
           <label>
```

Comment:
could simply add an `ng-if='vm.settings.destinyVersion === 2'` here so it only shows up when you're on the D2 version.

Reply:
I initially did that, but when I switch from d2 to d1 the setting pane will not have a selected search option, as the underlying setting is the same.

Not sure if that's preferable to just making the label "base power (destiny 2 only)" or maybe "base power (sorts on primary stat in destiny 1)".

Reply:
@kyleshay and @duckmanBR any opinion on this?

I think I would prefer to leave it as is, as I feel it implies the default option (primary stat) will be used on destiny 1. If you prefer to go another way I'm totally ok with changing it though, just wanna get the feature out there :D

Oh and sorry for nagging you, just excited to get this merged :)